src/app/portlets/content-types/edit/content-types-edit.component.html

```diff
@@ -7,48 +7,46 @@
             <div class="content-type__edit-title">
                 <h4>{{ data.name }} </h4>
                 <div class="content-type__edit-subtitle">
-                    <span class="content-type__edit-variable" *ngIf="isEditMode()">
+                    <span class="content-type__edit-variable">
                         {{ dotMessageService.get('contenttypes.content.variable') }}: {{ data.variable }} 
                         <span class="content-type__edit-dot-separator">•</span> {{ dotMessageService.get('contenttypes.form.identifier') }}: {{ data.id }}
                     </span>
                 </div>
             </div>
         </div>
-        <div *ngIf="isEditMode()">
-            <div class="content-type__edit-actions">
-                <button type="button" pButton flat label="Edit" (click)="editForm()" icon="fa-edit" id="form-edit-button"></button>
-            </div>
+        <div class="content-type__edit-actions">
+            <button type="button" pButton flat label="Edit" (click)="show = true" icon="fa-edit" id="form-edit-button"></button>
         </div>
     </div>
-    <p-dialog
-        modal="modal"
-        width="500"
-        [dismissableMask]="true"
-        [(visible)]="show"
-        [responsive]="true"
-        [resizable]="false"
-        [draggable]="false"
-        [closable]="isEditMode()"
-        [height]="dialogHeight"
-        (onHide)="closeDialog()"> 
-
-        <p-header class="container-fluid">
-            {{templateInfo.header}}
-        </p-header>
-
-        <dot-content-types-form
-            #form
-            [fields]="fields"
-            [data]="data"
-            (onSubmit)="handleFormSubmit($event)"
-            (closeDialog)="closeDialog()">
-        </dot-content-types-form>
-    </p-dialog>
-
     <dot-content-type-fields-drop-zone
         *ngIf="data.id"
         [fields]="fields"
         (saveFields)="saveFields($event)"
         (removeFields)="removeFields($event)">
     </dot-content-type-fields-drop-zone>
 </dot-content-type-layout>
+
+<p-dialog
+    *ngIf="show"
+    modal="modal"
+    width="500"
+    [(visible)]="show"
+    [responsive]="true"
+    [resizable]="false"
+    [draggable]="false"
+    [closable]="isEditMode()"> 
+    
+    <p-header>{{templateInfo.header}}</p-header>
+
+    <dot-content-types-form
+        #form
+        [fields]="fields"
+        [data]="data"
+        (onSubmit)="handleFormSubmit($event)">
+    </dot-content-types-form>
+
+    <p-footer>
+        <button type="button" pButton label="Cancel" class="content-type__cancel cancel" (click)="cancelForm()"></button>
```

Comment:
ALWAYS add ```class="cancel"``` to all the cancel buttons in dialogs, it make it look black, we don't want them blue because is like a "neutral" action.